pkg/tbtc/node.go

```diff
@@ -5,6 +5,7 @@ import (
 	"crypto/ecdsa"
 	"encoding/hex"
 	"fmt"
+	"go.uber.org/zap"
```

Comment:
Where does this dependency come from in our codebase?

Reply:
It's the actual logger implementation used by `github.com/ipfs/go-log/v2`